client.go

```diff
@@ -0,0 +1,32 @@
+package dp_redis
+
+import (
+	"github.com/go-redis/redis"
+	"time"
+)
+
+// Redis - structure for the redis client
+type Redis struct {
+	client *redis.Client
+	ttl    time.Duration
+}
+
+// Options - config options for the redis client
+type Options struct {
+	Addr     string
+	Password string
+	DB       int
```

Comment:
What is `DB`?

Reply:
Fair comment. It is database, `DB` is what it is called in the redis package I have imported too.